src/transport/SecureSessionTable.cpp

```diff
@@ -183,6 +220,100 @@ SecureSession * SecureSessionTable::EvictAndAllocate(uint16_t localSessionId, Se
 
 void SecureSessionTable::DefaultEvictionPolicy(EvictionPolicyContext & evictionContext)
 {
+    evictionContext.Sort([&evictionContext](const SortableSession & a, const SortableSession & b) {
+        //
+        // Sorting on Key1
+        //
+        if (a.mNumMatchingOnFabric > b.mNumMatchingOnFabric)
+        {
+            return true;
+        }
+
+        if (a.mNumMatchingOnFabric == b.mNumMatchingOnFabric)
+        {
+            int ScoreA = a.mSession->GetPeer().GetFabricIndex() == evictionContext.GetSessionEvictionHint().GetFabricIndex();
```

Comment:
start variable names with lowercase. mauybe cnst as well.

Why is this an int when the comparison is an equality. Naming of 'score' seems odd.

```cpp
const bool isEvictionHintA = ....;
const bool isEvictionHintB = ....;

if (isEvictionHintA != isEvictionHintB) {
   return isEvictionHintA;
}